site/migrations/000-create-users.js

```diff
@@ -6,7 +6,7 @@ exports.up = DB => {
 			name character varying(255),
 			username character varying(255) not null,
 			avatar text,
-			github_token character varying(255) not null,
+			github_token character varying(255),
```

Comment:
This should be the only change to the production DB required

Reply:
Will do

Reply:
Done